codebuild-terraform.tf

```diff
@@ -23,6 +23,31 @@ resource "aws_codebuild_project" "terraform_plan" {
       name  = "TF_IN_AUTOMATION"
       value = "true"
     }
+    dynamic "environment_variable" {
```

Comment:
Please add line breaks for readability between `}` and `dynamic` on each instance in this file

Reply:
I followed the original format. We can open a new PR with all the style changes.